dataland-frontend/tests/e2e/specs/user-api-key/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * As a user I want to be able to use the swagger UI to send requests to the backend
+ */
+describe("User api key tests", () => {
+  require("./GenerateApiKey");
+  require("./RevokeApiKey");
+  require("./DropDownMenuApiKeyElement");
```

Comment:
I think you need to add the new Tests (`CreateApiKeyCard.cy.ts` and `ApiKeyPage.cy.ts`) here.

Reply:
as discussed not needed as those are component tests